recipes/dav1d/all/test_package/conanfile.py

```diff
@@ -4,14 +4,14 @@
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake"
+    generators = "cmake", "cmake_find_package_multi"
```

Comment:
Don't use the `cmake_find_package` generator here and in the test package's CMakeLists.txt.
We only test functionality that the upstream project provides.
In this case, the upstream project only provides `.pc` files, no `.cmake` scripts.

Reply:
Done, forgot about it